test/Dialect/Moore/basic.mlir

```diff
@@ -25,6 +52,10 @@ moore.module @Foo {
 
   // CHECK: moore.assign %v1, %v2 : !moore.bit
   moore.assign %v1, %v2 : !moore.bit
+  // CHECK: moore.assign %w0, %w11 : !moore.logic
+  moore.assign %w0, %w11 : !moore.logic
+  // CHECK: moore.assign %w0, %w12 : !moore.logic
+  moore.assign %w0, %w12 : !moore.logic
```

Comment:
```suggestion
```
I don't think you need these, because the parser for `moore.assign` doesn't really know where `%v1` and `%v2` come from, and so the fact that `%v1` comes from `moore.variable` and `%w0` comes from `moore.net` doesn't really affect it, or cover any new code paths.